components/SVGMapLL.tsx

```diff
@@ -1,7 +1,15 @@
 import React from "react";
 import Svg, { G, Path, Text, TSpan } from "react-native-svg";
+import { getHightLightColorForShop } from "../helpers/hightlighting";
+import { SHOP_LIST } from "../constants/constants";
+import { SvgComponentProps as Props } from '../types/types';
 
-const SvgComponent = ({ navigateToShopId }) => (
+// 2do: Fix issue with TS complaining on typing of Text component from
```

Comment:
Please can you use 'TODO', this follow conventions in many IDEs and also opens up plugins -> https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight